content/docs/user-guide/data-management/remote-storage/amazon-s3.md

```diff
@@ -79,7 +79,7 @@ $ dvc remote modify --local myremote \
 $ dvc remote modify --local myremote \
                     credentialpath 'path/to/credentials'
 # and (optional)
-$ dvc remote modify --local myremote profile 'myprofile'
+$ dvc remote modify --local myremote profile 'mysecret'
```

Comment:
Why this change? A profile name shouldn't be a secret.

Reply:
If it's not a secret we don't need `--local` either. Up to you:

```suggestion
$ dvc remote modify myremote profile 'myprofile'
```

Reply:
Sure, I would do that. But I believe in general the recommendation is to be extra paranoid and don't even share that info. Anyway, 👍🏼